### ISSUE TITLE
HorseBuff Causing Lag

### ISSUE THREAD

**Issue by muhammedesadcomert** [CodeF53/Horse-Buff#42]

**Comment by CodeF53** [comment]
Have you tried removing it to make sure it's actually horsebuff?

HorseBuff doesn't touch any networking or packet code, which is what the errors are mentioning.

**Comment by muhammedesadcomert** [comment]
Of course, I tried it, and the lag is gone. I haven't had enough time yet to test it in an isolated modpack. It may be a mod incompatibility issue.

**Comment by allanonmage** [comment]
Did you ever track down what the lag was about?

**Comment by CodeF53** [comment]
I reformatted my pc and I haven't bothered to set up my mod development environment since then.

I haven't played minecraft in months, so I don't have much motivation to update my mods.

**Comment by muhammedesadcomert** [comment]
> I reformatted my pc and I haven't bothered to set up my mod development environment since then.
> 
> I haven't played minecraft in months, so I don't have much motivation to update my mods.

Are you open for pull requests? I would like to contribute this mod.

**Comment by CodeF53** [comment]
> Are you open for pull requests? I would like to contribute this mod.

I am absolutely open to pull requests! 

Only request is that you enable the github actions on your fork so I don't have to fuss around with git inorder to test your code.

**Comment by muhammedesadcomert** [comment]
Ok, that's cool!

Workflows will be run.

**Comment by allanonmage** [comment]
I was sad when you said you were abandoning it, but now I'm happy that someone will update it.  I've been trying to be particular about mods I'm adding to a modpack to make sure they will get updates.

**Comment by Sandr-aa** [comment]
I came to have come across the same problem, disabling `Fix Rubberbanding on Stairs` is an workaround

**Comment by somykOS** [comment]
mod version 2.1.8, still has this issue.
thanks to @Sandr-aa for commenting how to fix this
